org.civicrm.reqangular/src/common/decorators/ui-bootstrap/uib-modal.decorator.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env amd */
+
+define([], function () {
+  'use strict';
+
+  return ['$delegate', '$document', function ($delegate, $document) {
+    var openFunction = $delegate.open;
+
+    $delegate.open = open;
+
+    /**
+     * Decorates uibModal to set prevent body from scrolling while a modal is open
+     * and revert to the original styles values when it is closed
+     *
+     * @param  {any}
+     * @return {Object} Modal Instance
+     */
+    function open () {
+      var lockScrollStyling = ';overflow: hidden;height: 100%;width: 100%;';
```

Comment:
```js
 var lockScrollStyle = 'overflow: hidden; height: 100%; width: 100%;';
```
The `;` should not be part of the variable content, just add it when you concatenate the strings